spec/helpers/application_helper_spec.rb

```diff
@@ -0,0 +1,211 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+describe ApplicationHelper do
+  describe "#creation_id_for_css_classes" do
+    context "when creation is ExternalWork" do
+      let(:external_work) { create(:external_work) }
+
+      it "returns string for exteral work" do
+        result = creation_id_for_css_classes(external_work)
```

Comment:
When calling helper methods in helper specs we should be [using the `helper` object](https://relishapp.com/rspec/rspec-rails/v/4-0/docs/helper-specs/helper-spec). It's not necessary in this case since these helpers don't depend on other helper modules, but it's good practice.

Reply:
I'll add it now, since that's not necessarily true of all the code in application_helper, which we might some day want to test.